stablehlo/relay/RelayOps.cpp

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2023 The StableHLO Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "stablehlo/relay/RelayOps.h"
+
+#include "stablehlo/reference/Element.h"
+
+namespace mlir {
+namespace stablehlo {
+namespace relay {
+
+Tensor addOp(const Tensor &lhs, const Tensor &rhs, ShapedType resultType) {
```

Comment:
I think you do not actually get what relay means. Please refer to Apache TVM knowledges for Relay IR